test/nextjs.test.js

```diff
@@ -106,7 +107,7 @@ describe(`NextJS Middleware Test: ${printPath("[test/nextjs.test.js]")}`, functi
 
             superTokensNextWrapper(
                 async (next) => {
-                    return SuperTokens.middleware()(request, response, next);
```

Comment:
Ideally you should not have to change these...

Reply:
👍🏼